src/features/status-bar/index.tsx

```diff
@@ -0,0 +1,27 @@
+import dynamic from 'next/dynamic';
+import { ErrorBoundary } from 'react-error-boundary';
+import { QueryErrorResetBoundary } from '@tanstack/react-query';
+
+const renderLoadingComponent = () => null;
+const renderErrorComponent = () => null;
+
+const StatusBarLazy = dynamic(() => import('@/features/status-bar/StatusBar'), {
+  loading: renderLoadingComponent,
+  ssr: false,
+});
+
+export const StatusBar = () => (
+  <QueryErrorResetBoundary>
+    {({ reset }) => (
+      <ErrorBoundary
+        fallbackRender={renderErrorComponent}
+        onError={error => {
+          console.log(error);
```

Comment:
Should this be pushed to Sentry or otherwise captured?

Reply:
Yep exactly. We're having sentry disabled atm but will add it back soon.